redash/query_runner/sqlite.py

```diff
@@ -83,7 +83,11 @@ def run_query(self, query, user):
             error = "Query cancelled by user."
             json_data = None
         except Exception as e:
-            raise sys.exc_info()[1], None, sys.exc_info()[2]
+            # handle unicode error message
+            err_class = sys.exc_info()[1].__class__
+            err_args = [arg.decode('utf-8') for arg in sys.exc_info()[1].args]
+            unicode_err = err_class(*err_args)
+            raise unicode_err, None, sys.exc_info()[2]
```

Comment:
What if we remove this whole `except Exception` block, will it resolve the issue?